docker/frontend/files/etc/copr/copr.conf

```diff
@@ -136,3 +136,9 @@ MAX_BUILD_TIMEOUT = 108000
 
 # Disable template caching
 CACHE_TYPE = "NullCache"
+
+USAGE_TREEMAP_TEAMS = {
+    "CPT": ["@copr", "@mock", "msuchy", "praiskup", "frostyx", "nikromen"],
```

Comment:
Just discrepancy in the way we handle @copr/PyPI, which is mosthe tly a place where Python team works. Not a big issue, but it might look like we are the most consuming consumers of our service.  So the grouping could be done on a per-project level (at least in the future).

Reply:
Or maybe we can move `@copr/PyPI` outside of the `@copr` namespace? We could then create an empty `@copr/PyPI` with no builds and specify a runtime dependency for the outside-of-copr project, to ensure that all the articles written about it still work?